.changeset/silly-humans-float.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
+---
+
+Adds 'sx' prop, renames 'appearance' prop, fixes docs rendering mistake
```

Comment:
```suggestion
FormControl:
* Add `sx` prop
* Rename `appearance` prop to `variant`
```

Reply:
Ohh I didn't know you could do returns in that message 👍